### PR TITLE
Jeremy/eng 997c past cancelled

### PR DESCRIPTION
**Pull request by jerminatorhits**
Built off of 997b, ideally review/merge it first: https://github.com/thebeetoken/beenest-web/pull/222

## Description
Why did you write this code?
don't show cancel button on `Past` and `Cancelled` trip cards
add user friendly cancel status text in red

## How to Test
- [ ] Visit /work/trips
- [ ] Click `Past` and `Cancelled`
- [ ] Verify functionality and presentation as based on the existing /trips page

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?
Styling

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/work/TripCard/TripCard.tsx [hunk 1]
```diff
@@ -64,21 +72,20 @@ const TripCard = ({ booking, category, handleModalAction }: Props) => {
...
               return (
                 <Col key={i} onClick={onClick} className={i !== arr.length - 1 ? 'u-ver-divider' : ''}>
-                  {href ? (
-                    <a href={href}>
-                      <h5 className="small font-weight-normal text-secondary mb-0">{label}</h5>
-                    </a>
-                  ) : (
-                    <h5 className="small c-pointer font-weight-normal text-secondary mb-0">{label}</h5>
-                  )}
+                  <a href={href || '#'}>
```
Comment (tc) [inline src/components/work/TripCard/TripCard.tsx]: can we move the `onClick` to the `a` tag instead of the `Col` tag?

**Comment by tc** [comment]
we need a `/trips` link in the header?

**Comment by jerminatorhits** [comment]
@tc done
